### PR TITLE
Allow link-local ipv6 gateway addresses

### PR DESCRIPTION
**Pull request by jovandeginste**
This is a common pattern used to simplify auto-configuration
of IPv6 Networks by using link-local addresses (such as `fe80::1`)
for any subnet.

Since any v6-enabled interface gets an ip in the `fe80::/64` range,
ips needs to be scoped with the interface (`%$devname`, eg. `%eth0`).

For reference, some places where this is used:

https://wiki.hetzner.de/index.php/Netzkonfiguration_CentOS/en
http://openbsd-archive.7691.n7.nabble.com/IPv6-gateway-fe80-1-needs-a-ping-to-work-td264877.html
http://www.spinics.net/lists/netdev/msg365398.html

### CODE REVIEW

File: provisioning_templates/snippet/kickstart_networking_setup.erb [hunk 1]
```diff
@@ -176,12 +176,12 @@ PEERDNS=no
 PEERROUTES=no
 DEFROUTE=no
 <% end -%>
-<% if @host.ip6.present? -%>
+<% if interface.ip6.present? -%>
```
Comment (jovandeginste) [inline provisioning_templates/snippet/kickstart_networking_setup.erb]: Should this also be in a separate PR?

Comment (mmoll) [reply provisioning_templates/snippet/kickstart_networking_setup.erb]: yes, please

**Comment by ekohl** [comment]
There's a merge conflict

**Comment by jovandeginste** [comment]
Merge conflict should be solved

File: provisioning_templates/snippet/kickstart_networking_setup.erb (hunk 1 above)
Comment (jovandeginste) [reply provisioning_templates/snippet/kickstart_networking_setup.erb]: done

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
merged, bedankt @jovandeginste!
